mis/add_binary.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ### 67. Add Binary
+# 
+# https://leetcode.com/problems/add-binary/
+# 
+# Given two binary strings, return their sum (also a binary string).
+# 
+# The input strings are both non-empty and contains only characters 1 or 0.
+# 
+# Example 1:
+# 
+# Input: a = "11", b = "1"
+# Output: "100"
+# Example 2:
+# 
+# Input: a = "1010", b = "1011"
+# Output: "10101"
+
+# In[18]:
+
+
+# swap a, and b so that a is always the shorter length string
+# reversed bit order in each string
+def add_two_bin_strings(a, b):
```

Comment:
@loantkvo could you please extend this function to do for any base `n` (where `n` < 16).

e.g.

def sum_numbers(a, b, base_n=2):
  """
 Returns the sum of `a + b` in `base_n`.
  """
  pass